### PR TITLE
Fix SearchByImageResponse creating

### PR DESCRIPTION
**Pull request by jrawska**

### CODE REVIEW

File: src/Response/SearchByFeatureResponse.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ private function __construct(array $uids)
     public static function fromArray(array $response): SearchByFeatureResponse
     {
         try {
-            $products = $response['products'];
+            $products = $response[0]['products'];
```
Comment (malarzm) [inline src/Response/SearchByFeatureResponse.php:23]: what about `[1]`, `[2]` et al? What's their purpose? Can we safely assume that `[0]` is what we always need?

Comment (malarzm) [reply src/Response/SearchByFeatureResponse.php:23]: Please ask WideEyes what the heck :)

**Review by malarzm — APPROVED** [review]

File: src/Response/SearchByFeatureResponse.php (hunk 1 above)
Comment (jrawska) [reply src/Response/SearchByFeatureResponse.php:23]: I've done that, I'm waiting for an answer.

Comment (lukasz-falda) [reply src/Response/SearchByFeatureResponse.php:23]: `answer`?

**Review by lukasz-falda — APPROVED** [review]

**Pull request merged**